### ISSUE TITLE
Add smtp client for admin notification of errors

### ISSUE THREAD

**Issue by johnk-** [gchiu/rebolbot#22]

**Comment by gchiu** [comment]
this should be easy enough.  We just sign up for a sendgrid account which doesn't need a dkim.  that's what I'm doing with rebolchat.me.

Or, how about it posts the error to a rebolchat.me rebolbot error topic??

**Comment by asampal** [comment]
Yeah, that's not a bad idea, posting to a topic. It'll appear in the chat too.

**Comment by gchiu** [comment]
Ahhh... don't want it in the chat as well 👎

**Comment by gchiu** [comment]
import <smtp>
import <send>
set-net [
	compkarori+bot@gmail.com "smtp.sendgrid.net:465"
	_
	"apikey"
	"..private..api..key..which..is..not..shown..here.."
	"rebol.info"
]

send/subject john@whereever.com error-message "Rebol Error"
